charts/reporter/templates/bootstrap-mongodb.yaml

```
@@ -0,0 +1,102 @@
+{{- if .Values.global.externalMongoDefinitions.enabled }}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: reporter-bootstrap-mongodb
+  namespace: {{ include "global.namespace" . }}
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)